gherkin-utils/java/src/test/java/io/cucumber/gherkin/utils/GherkinPrettyFormatterTest.java

```diff
@@ -0,0 +1,112 @@
+package io.cucumber.gherkin.utils;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class GherkinPrettyFormatterTest {
```

Comment:
With Junit 5 test classes no longer have to be public. Please make them package private.

Reply:
Ok I've already changed it.